CFC_WebApp/main/tripManager.py

```diff
@@ -87,6 +41,13 @@ def stripoutNonSerializable(sectionList):
         strippedList.append(section)
     return strippedList
 
+def filter_unclassifiedSections(UnclassifiedSections):
+    filtered_Sections=[]
+    for section in UnclassifiedSections:
+        if section['filtered']:
```

Comment:
I have four comments here:

1. This check doesn't match the spec. Here, if section['filtered'] == True, then we retain the section, but the spec says:
> In the queryUnclassifiedSections we filter out any section entry that is labeled with section['filter'] = True

2. The code that sets the `'filtered'` field is also consistent with this description, and not with the spec. For example, in `label_filtered_section`, if the `distance > = minimum_travel_time`, we set `is_filtered = True`, not False. However, in this case, I think that maybe a different label name should have been used to minimize confusion.
1. if we are only filtering based on a simple flag, there is no need to read all the sections and iterate through them. Instead, this can be incorporated into the query for the sections, which should improve performance.
1. Are you sure that the unit test for this code works? There are three test sections in your file, only two of which are of type `move`, and both of them have `confirmed_mode` set. I would assume that all three of the sections would be filtered out by the `defaultQueryList`, and the test would fail.